Lib/turtledemo/colormixer.py

```diff
@@ -1,5 +1,7 @@
-# colormixer
+"""     turtle-example-suite:
 
+       turtledemo/colormixer.py
```

Comment:
I would like all of these 3-line headers condensed to 

```suggestion
"""turtledemo/colormixer.py
```
This directory is the turtledemo suite and saying this twice is redundant.

Reply:
This is not resolved because it was done only for colormixer, not all files.  Please let me (and other coredevs) decide if our comments are resolved.

Reply:
Thanks for informing, I'll be aware of that in the future